intelmq/tests/bots/experts/rdap/test_data/rdns.json

```diff
@@ -0,0 +1,6841 @@
+{
+    "description": "RDAP bootstrap file for Domain Name System registrations",
+    "publication": "2021-05-11T21:00:02Z",
+    "services": [
+      [
+        [
+          "nowruz",
```

Comment:
Do we really need the complete file with nearly 7k lines in the test data?

Reply:
No, we dont need it. But I copied it 1 by 1, gonna crop it :)